easybuild/easyblocks/b/boost.py

```diff
@@ -227,7 +227,15 @@ def install_step(self):
         """Install Boost by copying file to install dir."""
 
         self.log.info("Copying %s to installation dir %s" % (self.objdir, self.installdir))
-        copy(glob.glob(os.path.join(self.objdir, '*')), self.installdir)
+        try:
+            copy(glob.glob(os.path.join(self.objdir, '*')), self.installdir)
+        except EasyBuildError as return_error:
```

Comment:
@ocaisa Rather than trying and letting it fail, we should just check which iteration is running when `multi_deps` is used, and only copy the Python libraries from the 2nd iteration onwards.

```python
if self.cfg['multi_deps'] and self.iter_idx > 0:
    # only copy Python libs
else:
    copy(glob.glob(os.path.join(self.objdir, '*')), self.installdir)
```

Also, the sanity check should probably be enhanced so it checks for both `libboost_python*` libraries?

Reply:
You mean the sanity check is not run independently for each iteration? I thought it was, other wise how can you check a python import?

Reply:
The sanity check is run for both iterations, my bad.